.eslintignore

```diff
@@ -12,7 +12,7 @@ build-system/tasks/visual-diff/snippets/*.js
 build-system/babel-plugins/**/fixtures/**/*.js
 build-system/babel-plugins/**/fixtures/**/*.mjs
 build-system/server/app-index/test/*.js
-examples/**
+examples/**/*.html
```

Comment:
Alternatively:

```suggestion
!examples/visual-tests/**/*.js
examples/**
```

And undo all changes outside `visual-tests`.

Reply:
I'm good with those changes remaining as they are.